rstream/client.py

```diff
@@ -151,6 +151,19 @@ def is_connection_alive(self) -> bool:
     def add_stream(self, stream: str):
         self._streams.append(stream)
 
+    async def send_publish_frame(self, frame: schema.Publish) -> None:
```

Comment:
Can we reuse the `send_frame` something like:
```python
 async def send_frame(self, frame: schema.Frame) -> None:
        logger.debug("Sending frame: %s", frame)
        assert self._conn
        try:
            if self.is_connection_alive():
                if frame.key == constants.Key.Publish:
                    await self._conn.write_frame_publish(frame)
                else:
                await self._conn.write_frame(frame)
 ```

instead of a new method ?

Reply:
Yeah I also tought about it, it is working  but the checker is not liking this conversion:
rstream/client.py:173: error: Argument 1 to "write_frame_publish" of "Connection" has incompatible type "Frame"; expected "Publish"

If we want we can allow it to pass with an exception # type: ignore but I'm not sure it is the right way